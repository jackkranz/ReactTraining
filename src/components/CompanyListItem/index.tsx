import React from 'react';

interface Props {
  name: string;
}
const CompanyListItem = (props: Props): JSX.Element => {
  return <h2>{props.name}</h2>;
};

export default CompanyListItem;
