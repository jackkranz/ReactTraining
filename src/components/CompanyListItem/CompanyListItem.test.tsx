import React from 'react';
import { shallow } from 'enzyme';
import CompanyListItem from '.';

describe('Rendering', () => {
  it('Matches Snapshot', () => {
    const props = { company: 'test company 123' };
    const component = shallow(<CompanyListItem {...props} />);
    expect(component).toMatchSnapshot();
  });
});
