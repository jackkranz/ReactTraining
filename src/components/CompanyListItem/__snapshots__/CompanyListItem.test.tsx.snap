// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Matches Snapshot 1`] = `
<div
  className="company-list-item"
>
  <h2>
    test company
  </h2>
</div>
`;
